common/setups/rasr/util.py

```diff
@@ -244,10 +251,15 @@ def __init__(
             }
         }
         ##################################################
+        test_recognition_args
```

Comment:
If you keep it this way, add comment that the default `None` leads to no test recognition
```suggestion
        :param test_recognition_args:
```